src/imports.rs

```diff
@@ -315,8 +322,16 @@ impl UseTree {
             visibility,
             attrs,
         };
+
+        let mut leading_modsep = false;
+
+        if context.config.edition() == Edition::Edition2018 {
+            if a.prefix.to_string().len() > 2 && &a.prefix.to_string()[0..2] == "::" {
+                leading_modsep = true;
+            }
+        }
```

Comment:
I prefer not to use a `mut` variable here. I think that we can simplify this by using `&&` operator instead.